src/cloud/release-notes/mcd-release-notes.md

```diff
@@ -16,6 +16,59 @@ The release notes include:
 -  {:.new}New features
 -  {:.fix}Fixes and improvements
 
+## v1.2.0
+*Release date: November 9, 2020*<br/>
+
+-  **Container updates–**
+
+   -  **PHP-FPM container**
+
+      -  {:.new}Added support for the gnupg PHP extension.<!--MCLOUD-5981-->
+
+   -  **Database container**
+
+      -  {:.fix}Fixed the database container health check by adding the required database password to the health check command.<!--MCLOUD-7122-->
+
+   -  **Elasticsearch container**
+
+      -  {:.new}Added support for Elasticsearch 7.9 for compatibility with upcoming Magento releases.<!--MCLOUD-7190-->
```

Comment:
We also added Elastic 7.7 https://jira.corp.magento.com/browse/MCLOUD-6219

Reply:
So for 1.1.2 release, we already had release note for 7.7 support. Do I need to include it again?

Reply:
Got it. Sorry. Looks like PR marked incorrectly with 1.2.0 label